source/extensions/filters/http/grpc_web/grpc_web_filter.cc

```diff
@@ -300,7 +300,7 @@ Http::FilterDataStatus GrpcWebFilter::encodeData(Buffer::Instance& data, bool en
   // The decoder always consumes and drains the given buffer. Incomplete data frame is buffered
   // inside the decoder.
   std::vector<Grpc::Frame> frames;
-  decoder_.decode(data, frames);
+  std::ignore = decoder_.decode(data, frames);
```

Comment:
Should these turn into TODOs or bugs?

Reply:
TODOs is probably better?

I am hoping to leave it to filter owner to make the call  @taoxuy @nareddyt @lizan